contents/handbook/product/metrics.md

```diff
@@ -17,6 +17,7 @@ The metric is defined [in this action][action]. The concept of the metric is use
 - Analyzing any insight. Analyzing means viewing for 10 seconds or more. Insights include: trends, funnels, paths, lifecycle, stickiness.
 - Analyzing a recording. Watching a recording for 10 seconds or more.
 - Analyzing a correlation analysis report. Analyzing means viewing for 10 seconds or more. 
+- Analyzing a dashboard. Analyzing means viewing a dashboard for 10 seconds or more. Viewing a dashboard does not fire "insight analyzed" events for its dashboard items.
```

Comment:
We'd need to instrument the component first to be sure - but it would be good to validate that the retention is also good for Discoveries if we include Dashboard Analyzed. (e.g.  [Dashboard Viewed](https://app.posthog.com/insights/VZlCXU6X/edit#filters=%7B%22insight%22%3A%22RETENTION%22%2C%22target_entity%22%3A%7B%22id%22%3A%22viewed%20dashboard%22%2C%22name%22%3A%22viewed%20dashboard%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%7D%2C%22returning_entity%22%3A%7B%22id%22%3A%225043%22%2C%22type%22%3A%22actions%22%2C%22name%22%3A%22App%20Pageview%20-%20Logged%20in%22%2C%22order%22%3A0%7D%2C%22period%22%3A%22Week%22%2C%22retention_type%22%3A%22retention_first_time%22%2C%22display%22%3A%22ActionsTable%22%2C%22properties%22%3A%5B%5D%2C%22filter_test_accounts%22%3Atrue%2C%22aggregation_group_type_index%22%3A0%7D))

Given we want to optimize for something that leads to high retention.

Reply:
Random feedback: if above = true, then I agree with this proposal :)

Reply:
In general terms, looks like by including dashboards, retention wouldn't look better or worse. Numbers taking 6-week retention average below. https://github.com/PostHog/posthog/pull/8433 instruments `dashboard analyzed` which will enable even better signals on dashboards. My proposal: let's track the current metric & potential new metric for ~2 weeks and see how numbers would change (both Discoveries numbers and retention).

**Baseline:** ~25.9% ([source](https://app.posthog.com/insights/VZlCXU6X/edit#filters=%7B%22insight%22%3A%22RETENTION%22%2C%22target_entity%22%3A%7B%22id%22%3A%2210784%22%2C%22name%22%3A%22Discoveries%22%2C%22type%22%3A%22actions%22%2C%22order%22%3A0%7D%2C%22returning_entity%22%3A%7B%22id%22%3A%2210784%22%2C%22type%22%3A%22actions%22%2C%22name%22%3A%22Discoveries%22%2C%22order%22%3A0%7D%2C%22period%22%3A%22Week%22%2C%22retention_type%22%3A%22retention_first_time%22%2C%22display%22%3A%22ActionsTable%22%2C%22properties%22%3A%5B%5D%2C%22filter_test_accounts%22%3Atrue%2C%22aggregation_group_type_index%22%3A0%7D)).
**Discoveries + Dashboards:** ~25.5% ([source](https://app.posthog.com/insights/VZlCXU6X/edit#filters=%7B%22insight%22%3A%22RETENTION%22%2C%22target_entity%22%3A%7B%22id%22%3A%2214561%22%2C%22name%22%3A%22Temp%3A%20Discoveries%20%2B%20Dashboards%22%2C%22type%22%3A%22actions%22%2C%22order%22%3A0%7D%2C%22returning_entity%22%3A%7B%22id%22%3A%2210784%22%2C%22type%22%3A%22actions%22%2C%22name%22%3A%22Discoveries%22%2C%22order%22%3A0%7D%2C%22period%22%3A%22Week%22%2C%22retention_type%22%3A%22retention_first_time%22%2C%22display%22%3A%22ActionsTable%22%2C%22properties%22%3A%5B%5D%2C%22filter_test_accounts%22%3Atrue%2C%22aggregation_group_type_index%22%3A0%7D )).

Reply:
Created [this dashboard](https://app.posthog.com/dashboard/29784) to start tracking performance of the new metric once https://github.com/PostHog/posthog/pull/8433 is merged.

Reply:
Sounds good to me.